requirements/fabric/strategies.txt

```diff
@@ -1,3 +1,3 @@
 # NOTE: the upper bound for the package version is only set for CI stability, and it is dropped while installing this package
 #  in case you want to preserve/enforce restrictions on the latest compatible version, add "strict" as an in-line comment
-deepspeed >=0.6.0, <=0.8.0; platform_system != "Windows"
+deepspeed >=0.6.2, <=0.8.0; platform_system != "Windows"
```

Comment:
Increased the min version because we need this fix: https://github.com/microsoft/DeepSpeed/commit/73b5d9833a113c47758fbe9f8538614ce34ea95e